integration_tests/e2e/induction/updatePreviousWorkExperienceDetail.cy.ts

```diff
@@ -0,0 +1,105 @@
+import Page from '../../pages/page'
+import WorkAndInterestsPage from '../../pages/overview/WorkAndInterestsPage'
+import { putRequestedFor } from '../../mockApis/wiremock/requestPatternBuilder'
+import { urlEqualTo } from '../../mockApis/wiremock/matchers/url'
+import { matchingJsonPath } from '../../mockApis/wiremock/matchers/content'
+import PreviousWorkExperienceDetailPage from '../../pages/induction/PreviousWorkExperienceDetailPage'
+import Error404Page from '../../pages/error404'
+
+context('Update previous work experience details in the Induction', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+    cy.task('stubUpdateInduction')
+    cy.task('stubGetInductionLongQuestionSet')
+    cy.signIn()
+  })
+
+  it('should update Previous Work Experience Detail given page submitted with changed job role and detail and no validation errors', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+
+    // stubGetInductionLongQuestionSet has previous work experiences of Office and Other
```

Comment:
Very minor typo (which we can address later if you prefer): should read `Office manager`, not `Office`

Reply:
No probs, I'll address it in my next PR 👍 , but its actually meant to be describing the list of types, rather than any specific role (because the test is based around a work experience type, by virtue of the path parameter), so I'll change it to:

```
// stubGetInductionLongQuestionSet has previous work experiences whose types are OFFICE and OTHER
```